arpav_ppcv/main.py

```diff
@@ -129,6 +132,23 @@ def run_server(ctx: typer.Context):
         uvicorn_args.extend(["--log-level=info"])
     if (log_config_file := settings.log_config_file) is not None:
         uvicorn_args.append(f"--log-config={str(log_config_file)}")
+
+    serving_str = (
+        f"[dim]Serving at:[/dim] [link]http://{settings.bind_host}:{settings.bind_port}[/link]\n\n"
+        f"[dim]Public URL:[/dim] [link]{settings.public_url}[/link]\n\n"
+        f"[dim]API docs:[/dim] [link]{settings.public_url}/docs[/link]"
+    )
+    panel = Panel(
+        (
+            f"{serving_str}\n\n"
+            f"[dim]Running in [b]{'development' if settings.debug else 'production'} mode[/b]"
+        ),
+        title="ARPAV-PPCV",
+        expand=False,
+        padding=(1, 2),
+        style="green",
+    )
+    print(Padding(panel, 1))
```

Comment:
This is just a little improvement I adapted from the [FastAPI CLI](https://github.com/tiangolo/fastapi-cli) project